migrations/20230223204539_002_label.js

```diff
@@ -0,0 +1,25 @@
+exports.up = function (knex) {
+    return knex("energizers").insert([
+        { name: "Red Bull", description: " Morning Energypiugpyifgyuvuyhvjhvjhbc EYGC liugqd  uigigas iugigf" },
+        { name: "Monster", description: "Rise and shine jhgaef POIUOIUJR ;UH iugf oih g ihi jh" },
+        { name: "5-hour Energy", description: "Energy Shot uiguif biuhoiuj yitg uitgwef ov fiu fr iuy" },
+        { name: "Rockstar", description: "Helloooo uef  oihoihwgf piuiuf  'oih'oh'ohf o';ouh;ef Energy Drink" },
+        { name: "Bang", description: "Hi gyuigwef honey ;uhf 'oihoihw ef iuh;ugfw Energy Drink" },
+    ])
+        .then(function () {
+            return knex("energizer_ratings").insert([
+                { energizer_id: 1, rating: 4 },
```

Comment:
This assumes the id that is being assigned to the rows above which might not be true. Can this is be rewritten to use the correct id returned by the database?